tests/continental_extension.prm

```diff
@@ -15,38 +15,16 @@ include $ASPECT_SOURCE_DIR/cookbooks/continental_extension/continental_extension
 
 set Timing output frequency                = 100
 set End time                               = 1e4
+set Max nonlinear iterations               = 1
```

Comment:
Isn't it possible to remove the temperature boundary conditions as well? I think they're the same as in the cookbook?

Reply:
Yep!